deno_dist/client/types.ts

```diff
@@ -26,10 +26,24 @@ type ClientRequest<S extends Data> = {
     : never
 }
 
-export interface ClientResponse<T> extends Response {
+export interface ClientResponse<T> {
+  ok: boolean
+  status: number
+  statusText: string
+  headers: Headers
+  url: string
+  redirect(url: string, status: number): Response
+  clone(): Response
   json(): Promise<T>
+  text(): Promise<string>
+  blob(): Promise<Blob>
+  formData(): Promise<FormData>
+  arrayBuffer(): Promise<ArrayBuffer>
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface Response extends ClientResponse<any> {}
```

Comment:
perhaps you could use `unknown` instead of `any` here? like:

`export interface Response extends ClientResponse<unknown> {}`

Reply:
You are right! It should be `unknown`.